core/src/main/python/wlsdeploy/util/string_utils.py

```diff
@@ -104,3 +107,56 @@ def load_properties(property_file, exception_type=None):
         prop_dict[key] = value
 
     return prop_dict
+
+
+def is_weblogic_version_or_above(wls_version, str_version):
+    """
+    Is the provided version number equal to or greater than the version encapsualted by this version instance
+    :param wls_version: the array representation of the current weblogic version to be compared
+    :param str_version: the string representation of the version to be compared
```

Comment:
I think both arguments are strings (both call .split()), also hard to describe. maybe:
```
wls_version: the string representation of the WebLogic version to be compared
str_version: the string representation of the WebLogic version to compare against
```
?

Reply:
Thanks changed.